ledger/test-common/src/main/daml/model/Test.daml

```diff
@@ -278,9 +273,11 @@ template BranchingControllers
     ctrlFalse : Party
   where
   signatory giver
-  controller (if whichCtrl then ctrlTrue else ctrlFalse) can
-    Delete : ()
-      do return ()
+  observer ctrl
+  let ctrl = if whichCtrl then ctrlTrue else ctrlFalse
```

Comment:
Is it syntactically valid to have the `let ctrl =` line above the `observer` line?

Reply:
yes moved it around